docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md

```diff
@@ -136,4 +132,4 @@ To customize the footer of a public dashboard, add the following section to the
 ;header_logo_hide =
 ```
 
-If you specify `footer_hide` to `true`, all the other values are ignored because the footer will not be shown.
+`footer_hide` is no longer available. Set either a `footer_text` or `footer_logo`. Otherwise, the default Grafana footer is displayed.
```

Comment:
I think you should add the `breaking-change` label to this PR and add this as the deprecation notice.

I don't think we should mention a removed option from the documentation so this should be replaced by: 
```
You can set either the `footer_text`, the `footer_logo`, or both. Otherwise, the default Grafana footer is displayed.
```

Reply:
How do I add deprecation notice? Is it same as breaking change notice in the pr ([ex](https://github.com/grafana/grafana-enterprise/pull/6328)?

Reply:
Also this is an enterprise only feature right? Should the notice/label be in the [enterprise pr](https://github.com/grafana/grafana-enterprise/pull/6235) instead?